src/unibasicTemplates/find.njk

```diff
@@ -23,14 +23,6 @@ if udoGetProperty(options, 'projection', projection, type) then
   go returnFromSub
 end
 
-* Temporary length check on the query since this is the query limit we are most likely to exceed.
-* STORIS Rocket case 00656416 - exceeding a query limit does not set an error condition in @system.return.code
-* TODO CXM-2318 - Add handling to mvom to prevent queries that will exceed db server limits
```

Comment:
There's another reference to CXM-2318 below (can't stick a comment there for some reason).

With this change we should be able to re-enable using the `returning` clause so we can return better error codes.

Reply:
It looks like it was using the `returning` statement previously but wasn't doing anything with the error code result.  I reverted it back to the way it was previously, but there might need to be additional work to do something with the error codes.